modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

```diff
@@ -0,0 +1,179 @@
++
+The following example configures a PVC that claims local persistent storage for Alertmanager:
++
+[source,yaml,subs=quotes]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    *alertmanagerMain*:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: *40Gi*
+----
+
+** *To configure a PVC for a component that monitors user-defined projects*:
+.. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
+
+.. Add your PVC configuration for the component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    <component>:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: <storage_class>
+          resources:
+            requests:
+              storage: <amount_of_storage>
+----
++
+See the link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[Kubernetes documentation on PersistentVolumeClaims] for information on how to specify `volumeClaimTemplate`.
++
+The following example configures a PVC that claims local persistent storage for the Prometheus instance that monitors user-defined projects:
++
+[source,yaml,subs=quotes]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    *prometheus*:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: *40Gi*
+----
++
+In the above example, the storage class created by the Local Storage Operator is called `local-storage`.
++
+The following example configures a PVC that claims local persistent storage for Thanos Ruler:
++
+[source,yaml,subs=quotes]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    *thanosRuler*:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: *40Gi*
+----
+
+. Save the file to apply the changes. The pods affected by the new configuration are restarted automatically and the new storage configuration is applied.
++
+[NOTE]
+====
+Configurations applied to the `user-workload-monitoring-config` `ConfigMap` object are not activated unless a cluster administrator has enabled monitoring for user-defined projects.
```

Comment:
This is already enabled for OSD/ROSA, so this note is not needed

Reply:
Removed.